08-generic_cache/02_routines/Makefile

```diff
@@ -7,6 +7,9 @@ build:
 test:
 	go test ./...
 
+testrace:
+	go test -race ./...
```

Comment:
Note, there's no point in having `go test -race` and regular `go test` separate. I would say, if you don't have a reason to remove `-race`, then keep it. It's an easy and relatively low-cost check to have and prevents major mistakes.